worker/worker.go

```diff
@@ -681,24 +680,8 @@ func (w *Worker) rhpFundHandler(jc jape.Context) {
 	ctx = WithGougingChecker(ctx, w.bus, gp)
 
 	// fund the account
-	jc.Check("couldn't fund account", w.withRevision(ctx, defaultRevisionFetchTimeout, rfr.ContractID, rfr.HostKey, rfr.SiamuxAddr, lockingPriorityFunding, func(rev types.FileContractRevision) (err error) {
-		h := w.Host(rfr.HostKey, rev.ParentID, rfr.SiamuxAddr)
-		err = h.FundAccount(ctx, rfr.Balance, &rev)
-		if isBalanceMaxExceeded(err) {
```

Comment:
Ignoring this error seems like something that would lead to a lot of spam and/or issues.
At the very least the autopilot would retry over and over again so we might want to extend `WithDeposit` to schedule a sync.